README.md

```diff
@@ -737,6 +738,24 @@ e.g:
 turbo crypto-fund --value 0.0001 --token kyve --private-key 'b27...45c'
 ```
 
+##### `upload-folder`
+
+Upload a folder of files to the Turbo Upload Service. This command will upload all files in the folder and create a manifest file for the folder.
+
+Command Options:
+
+- `-f, --folder-path <folderPath>` - Path to the folder to upload
+- `--index-file <indexFile>` - Index file to use in the manifest created for folder upload
+- `--fallback-file <fallbackFile>` - Fallback file to use in the manifest created for folder upload
```

Comment:
Might want to update the comment slightly to something like `File to use for the "fallback" path in the resulting manifest`